kazaam_benchmarks_test.go

```diff
@@ -270,6 +270,34 @@ func BenchmarkExtractTransformOnly(b *testing.B) {
 
 }
 
+func BenchmarkTimestamp(b *testing.B) {
+	b.ReportAllocs()
+
+	spec := `[{"operation": "timestamp", "spec": {"ops":[{"ops":"notArrayKey","inputFormat":"Mon Jan _2 15:04:05 -0700 2006","outputFormat":"2006-01-02T15:04:05-0700}]}}]`
```

Comment:
Maybe I am missing something here, what is this `ops` key in the spec? Why is it not like `"spec": {"topKeyA.notArrayKey":{"inputFormat":"Mon Jan _2 15:04:05 -0700 2006","outputFormat":"2006-01-02T15:04:05-0700"}}`?